core/src/test/java/org/hibernate/ogm/backendtck/queries/CompositeIdQueriesTest.java

```diff
@@ -88,7 +88,8 @@ public void testSingleAttributeProjection() throws Exception {
 
 	@Test
 	public void testProjections() throws Exception {
-		List<ProjectionResult> result = asProjectionResults( session, "SELECT n.newsId.title, n.newsId.author, n.content FROM News n WHERE n.newsId.title = '" + ogmHowTo.getNewsId().getTitle() + "'" );
+		List<ProjectionResult> result = asProjectionResults(
+				session, "SELECT n.newsId.title, n.newsId.author, n.content FROM News n WHERE n.newsId.title = '" + ogmHowTo.getNewsId().getTitle() + "'" );
```

Comment:
Did you set the code auto-format? there several indentation that have changed. It's not critical but, please, refrain from doing it if you did it on purpose